services/analytics/analytics.ts

```
@@ -1,28 +1,74 @@
+export class Analytics {
+  private static readonly MAX_EVENT_VALUE_LENGTH = 100;
 
-  pageview: (path: string, title?: string) => {
-    // Pageviews are automatically tracked by the GoogleAnalytics component
-    // This method is kept for backward compatibility
-  },
+  static event(category: string, action: string, value?: string) {
+    try {
+      // Sanitize and truncate the value to prevent sensitive data leaks
+      const sanitizedValue = value ? 
+        value.substring(0, this.MAX_EVENT_VALUE_LENGTH) : undefined;
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)